.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,36 +4,55 @@ about: Create a report to help us improve
 
 ---
 
-<!--
-PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.
+<!-- Before submitting an issue, please consult our troubleshooting guide (http://ionicframework.com/docs/troubleshooting/) and developer resources (http://ionicframework.com/docs/developer-resources/) -->
 
-ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION.

```

Comment:
I'm not sure that we should remove this, we want to make it known that if the template is ignored we may close the issue as missing template

Reply:
I added add in line 13